packages/nvidia_nat_app/tests/executors/test_runner.py

```
@@ -0,0 +1,426 @@
+
+        async def run_node(name: str):
+            if name == "router":
+                await asyncio.sleep(0.02)
+                return {"decision": "left"}
+            return await _slow_node(name)
+
+        result = await run_speculation(
+            plan,
+            state,
+            run_node=run_node,
+            get_decision=lambda r: "left",
```

Ruff (0.15.2)

[warning] Unused lambda argument: `r`

(ARG005)